scripts/deploy_oak_functions_api

```diff
@@ -20,7 +20,7 @@ readonly OUT_DIR='./oak_functions/bin'
 mkdir --parents "${OUT_DIR}"
 
 cp \
-  ./oak_functions/proto/server.proto \
+  ./remote_attestation/proto/remote_attestation.proto \
```

Comment:
I am expecting that the content of the remote attestation proto would move into oak_functions/proto/server.proto. Effectively that is the new interface to Oak Functions from now on.

Reply:
Created #2141, but I am not sure why would `remote_attestation` have a dependency to `oak_functions`. Is remote attestation specific to Oak Functions? My understanding, from the discussion on Slack, was that `remote_attestation` is going to be the more general handshake protocol.